.github/workflows/license-check.yml

```
@@ -0,0 +1,40 @@
+name: Check licenses
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  license_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.14.2'
```

I would use the option that uses the Node version specified in the _.nvmrc_ file. This will require using a newer version of the action. `v4` is the newest, but our other repos have only been upgraded to `v3`, so I don't know if `v4` breaks anything.
```suggestion
      - name: Set up Node
        uses: actions/setup-node@v4
        with:
          node-version-file: '.nvmrc'
```